src/pkm/PK3RR.ts

```diff
@@ -0,0 +1,371 @@
+// This file was generated by make generate
```

Comment:
I'd get rid of this comment, the reason it's present in the other files is because I made a tool to generate TypeScript code from JSON "schema files". I'm going to remove the comment from the other files soon because the tool has gotten too convoluted for its own good